tests/vm_test.go

```diff
@@ -1779,6 +1812,82 @@ var _ = Describe("[rfe_id:1177][crit:medium][vendor:cnv-qe@redhat.com][level:com
 					Entry("[test_id:6993]VMI launcher pod compute container should keep running", "true"),
 				)
 			})
+
+			Context("Using expand command", func() {
+				BeforeEach(func() {
+					newVM = newVirtualMachine(true)
+				})
+
+				It("should fail without arguments", func() {
+					expandCommand := clientcmd.NewRepeatableVirtctlCommand(vm.COMMAND_EXPAND)
+					err = expandCommand()
+
+					Expect(err).To(HaveOccurred())
+					Expect(err).Should(MatchError("error invalid arguments - Virtual Machine name or file must be provided"))
+				})
+
+				It("should expand vm", func() {
+					expandCommand := clientcmd.NewRepeatableVirtctlCommand(vm.COMMAND_EXPAND, "--namespace", newVM.Namespace, newVM.Name)
```

Comment:
Better use `util.NamespaceTestDefault` as namespace during functional tests if not otherwise required.